src/applications/accreditation/21a/pages/02-military-service-chapter/militaryServicePages.js

```diff
@@ -13,13 +14,18 @@ import {
 } from '~/platform/forms-system/src/js/web-component-patterns';
 
 import MilitaryServiceIntro from '../../components/02-military-service-chapter/MilitaryServiceIntro';
-import { formatReviewDate } from '../helpers/formatReviewDate';
-
 import {
   branchOptions,
   characterOfDischargeOptions,
   explanationRequired,
 } from '../../constants/options';
+import { formatReviewDate } from '../helpers/formatReviewDate';
+
+const getDateRange = item => {
```

Comment:
should this move to helpers? Seems like it could be useful for the employment "I still work here" as well